src/component/admin/auth/AdminLogin.js

```diff
@@ -0,0 +1,133 @@
+import {
+    Box,
+    Button,
+    FormControl,
+    Input,
+    Link,
+    Menu,
+    MenuButton,
+    MenuItem,
+    MenuList,
+    Text,
+    VStack
+} from "@chakra-ui/react";
+import {useState} from "react";
+import GeneralButton from "../../reusables/GeneralButton";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faEye } from "@fortawesome/free-solid-svg-icons";
+import axios from "axios";
+import {TriangleDownIcon} from "@chakra-ui/icons";
+const eye = <FontAwesomeIcon bg={"white"} h={'1.7vh'} w={'2vh'} icon={faEye} />;
+
+
+const AdminLogin = () => {
+    const[account, setAccount] = useState('');
+    const[email, setEmail] = useState('');
+    const[phone, setPhone] = useState('');
+    const[password, setPassword] = useState('');
+
+
+
+    const [passwordShown, setPasswordShown] = useState(false);
+
+    const togglePasswordVisibility = () => {
+        setPasswordShown(!passwordShown);
+    };
+
+    const onSubmit = (e) => {
+        e.preventDefault();
+        // localStorage.setItem('')
+        axios.post('http://localhost:8080/api/customer', this.state)
```

Comment:
There should be error handling at this point in case the request fails,
There should also be a way of notifying the user that the login attempt was successful
Look into the toastify library.

Reply:
What happens if the user logs in? ,
You are just logging the response, Ideally on a successful login, the user should be redirected to another part of the application

Reply:
You are hard coding the url directly here, and this makes the code base harder to maintain.
You should store the URLs in a variable and reference that variable whenever you want to use it.
That way you can easily make changes in one place and it'll take effect everywhere.